src/main/java/de/tum/in/www1/artemis/service/SubmissionVersionService.java

```diff
@@ -80,7 +80,7 @@ private SubmissionVersion createNewVersion(Submission submission, User user) {
 
     private String getSubmissionContent(Submission submission) {
         if (submission instanceof ModelingSubmission) {
-            return ((ModelingSubmission) submission).getModel();
+            return ("Model: " + ((ModelingSubmission) submission).getModel() + "; Explanation: " + ((ModelingSubmission) submission).getExplanationText());
```

Comment:
can you add new line here? This would make it easier to read and parse in case this is needed

Reply:
Sure. I added it.